### ISSUE TITLE
Git repo is very large

### ISSUE THREAD

**Issue by thescarydoor1** [tubbo/openrct2-benchwarmer#115]
Cloning the repo was a 1.73 GB download, this suggests that git is tracking large binary files

I don't really know much about yarn but I'm suspecting that tracking the yarn cache in the repo is what is causing the bloat

Not a big deal but just thought you should know

![Capture](https://github.com/user-attachments/assets/3d882691-3876-46c2-9d08-b7eb584ecfe9)

**Comment by tubbo** [comment]
Yeah I should probably upgrade that because I’m not sure that’s still the way to do “zero installs” anymore. The idea being that you don’t need to run yarn install as the deps come with the repo.
  

  
  
  
>

**Comment by fnielson1** [comment]
I definitely prefer running `yarn install`. If I have a PR with the binaries removed (and everything works), would you be willing to accept the change?
